cmd/storj-sim/network.go

```diff
@@ -148,17 +148,22 @@ func newNetwork(flags *Flags) (*Processes, error) {
 	processes := NewProcesses(flags.Directory)
 
 	var (
-		host                   = flags.Host
-		gatewayPort            = 9000
-		bootstrapPort          = 9999
-		bootstrapPrivatePort   = 9988
-		satellitePort          = 10000
-		satellitePrivatePort   = 11000
-		storageNodePort        = 12000
-		storageNodePrivatePort = 13000
-		consolePort            = 10100
-		bootstrapWebPort       = 10010
-		versioncontrolPort     = 10011
+		host                    = flags.Host
+		gatewayPort             = 9000
+		gatewayDebugPort        = 9500
+		bootstrapPort           = 9999
+		bootstrapPrivatePort    = 9988
+		bootstrapDebugPort      = 9977
+		satellitePort           = 10000
+		satellitePrivatePort    = 11000
+		satelliteDebugPort      = 11500
+		storageNodePort         = 12000
+		storageNodePrivatePort  = 13000
+		storageNodeDebugPort    = 13500
```

Comment:
Port 14500 for the storage nodes would be better. Port 13500 will allow us to run only 500 storage nodes. I don't know if I will run a storj-sim network with more than 500 storage nodes but I would like to have the option if needed.